### ISSUE TITLE
Accessibility: color attributes

### ISSUE THREAD

**Issue by dougmet** [datasciencecampus/sdg-indicators#567]
If you set any of the colors on the BODY or A elements you must set all of them.

> In HTML the color attributes are TEXT, BGCOLOR, LINK, ALINK and VLINK. In CSS the attributes are COLOR and BACKGROUND-COLOR. Some users have browser defaults set to white text on a black background, so setting one color without setting the others can result in black text on a black background. High contrast mode on Windows turns off background images so you must set a background-color in addition to a background-image.

https://datasciencecampus.github.io/assets/css/style.css?v=7626f0c006a9834988aae5c9fffef51c82a26fdb line 105 106 
https://datasciencecampus.github.io/sdg-indicators/assets/css/default.css?v=20171010134536 line 4  
	Guideline: WCAG 2.0 AA F24 Section 508 (2017) AA F24

**Comment by dougmet** [comment]
Actually this doesn't seem so bad. So I think we need to add colours here https://github.com/datasciencecampus/sdg-indicators/blob/3474a4a4841bd53328ea1690a43892827912305f/assets/css/default.scss#L37 or maybe we just need to set a default for `<a>` tags?

**Comment by dougmet** [comment]
Reading it I think it's background colours for all the `<a>` tags and maybe `<body>` too. That's my guess.

**Comment by gingerbenw** [comment]
body tag already had a colour set, I have now set a default for <a>, which uses a variable set at the top of the css. 

I can't find any backgrounds which are currently set as an image without also having a colour set, which is good!

**Comment by gingerbenw** [comment]
![screen shot 2017-10-26 at 16 07 09](https://user-images.githubusercontent.com/24570483/32060816-cd268bc0-ba67-11e7-8e83-a48696d32b97.png)

High contrast mode tested - all appears to be functioning

**Comment by gingerbenw** [comment]
![screen shot 2017-10-26 at 16 13 31](https://user-images.githubusercontent.com/24570483/32061101-a62ce810-ba68-11e7-85a5-999892ee268f.png)

It appears background colors are also disabled in high contrast mode

**Comment by dougmet** [comment]
Hi @gingerbenw, OK great. I think we should refer our accessibility testers here and if there's still an issue then we'll target it more directly.
